onnxruntime/test/qnn_ctx_gen/README.md

```diff
@@ -0,0 +1,31 @@
+# ONNXRuntime Qnn Context Generator
```

Comment:
Perhaps it would be good to update the README with basic information on how an application would use the generated models. It wasn't immediately obvious to me that the application only needs to set `ep.share_ep_contexts` to 1, but would set `enable_htp_weight_sharing` to 0.

Reply:
I will create separate session in Ort doc branch to describe the whole workflow.